### ISSUE TITLE
Carthage built magical record framework version shown as AppStore_1.3.1 in Info.plist

### ISSUE THREAD

**Issue by tharindu** [magicalpanda/MagicalRecord#1274]
Looks like building MagicalRecord via Carthage and submitting apps have following error.
<img width="661" alt="error" src="https://cloud.githubusercontent.com/assets/84404/20476610/decbe9f4-aff7-11e6-9d1d-a03c79ff72d0.png">
<img width="796" alt="error_2" src="https://cloud.githubusercontent.com/assets/84404/20476622/f7468e44-aff7-11e6-91ba-e6ddd7f60bc9.png">

**Comment by Coeur** [comment]
Is it still happening with MagicalRecord 2.3.3 or 2.4.0 and latest Carthage?

**Comment by tharindu** [comment]
Yes. It still shows the last tag name of my personal repository in Magical Record framework's `Info.plist` Updated to latest version and tested too. So I have to manually update the Info.plist files version into a valid number after `carthage update --platform iOS`

**Comment by suniljakhar** [comment]
Is this issue still exist?

**Comment by Coeur** [comment]
@suniljakhar I'm not a Carthage user, so you'll have to make a pull request eventually.
